pip_review/__main__.py

```diff
@@ -244,55 +136,59 @@ def confirm(question):
     return answer == 'y'
 
 
+def get_outdated_packages(local=False, pre_release=False, user=False):
+    command = ['pip', 'list', '--outdated', '--disable-pip-version-check']
+    if local:
+        command.append('--local')
+    if pre_release:
+        command.append('--pre')
+    if user:
+        command.append('--user')
```

Comment:
I really believe that we should not explicitly list the above three flags (and `--editable`). Instead, we should be forwarding any "unexpected" arguments directly to `pip list`. If you need clarification or if you think this is too complicated, let me know.